src/main/java/com/thealgorithms/ciphers/DES.java

```diff
@@ -169,18 +169,18 @@ private String encryptBlock(String message, String[] keys) {
         for (i = 0; i < 64; i++) {
             permutedMessage.append(message.charAt(IP[i] - 1));
         }
-        String L0 = permutedMessage.substring(0, 32);
-        String R0 = permutedMessage.substring(32);
+        String e0 = permutedMessage.substring(0, 32);
+        String f0 = permutedMessage.substring(32);
```

Comment:
Is there a reason why you changed from `L0` to `e0` etc?

Reply:
could change to l0 but e0 and f0 have better readability.